build/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.4
+
+- Use "strict raw types".
```

Comment:
worth a changelog entry?

Reply:
🤷‍♂️ I was thinking I'd publish to flush out any problems that this may cause so we know soon if we need to roll back. I don't expect any problems since this works internally...

Reply:
We should have a changelog entry for every version no matter what, so either this or `internal cleanup` would be fine.